File: app/src/main/java/com/zhang/mydemo/kotlin/ui/widgetkt/MyView.java
package com.zhang.mydemo.kotlin.ui.widgetkt;

import android.content.Context;
import android.graphics.Path;
import android.util.AttributeSet;
import android.view.View;

import androidx.annotation.Nullable;

/**
 * @Author : zhang
 * @Create Time : 2022/1/25
 * @Class Describe : 描述
 * @Project Name : MyDemo
 */
public class MyView  extends View {

    public MyView(Context context) {
        super(context,null);
    }

    public MyView(Context context, @Nullable AttributeSet attrs) {
        super(context, attrs,0);
    }

    public MyView(Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }
}

File: app/src/main/java/com/zhang/mydemo/ui/widget/MyView.kt
package com.zhang.mydemo.ui.widget

import android.content.Context
import android.util.AttributeSet
import android.view.View

/**
 * @Author : zhang
 * @Create Time : 2022/1/25
 * @Class Describe : 描述
 * @Project Name : MyDemo
 */
class MyView : View {
    constructor(context: Context?) : super(context, null) {}
    constructor(context: Context?, attrs: AttributeSet?) : super(context, attrs, 0) {}
    constructor(context: Context?, attrs: AttributeSet?, defStyleAttr: Int) : super(
        context,
        attrs,
        defStyleAttr
    ) {
    }
}